### DISCUSSION TITLE
#opensourcefriday AI and OSS

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [community/community#123787] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

This [#OpenSourceFriday](https://twitter.com/hashtag/OpenSourceFriday?src=hash), join [@dwcheatham](https://twitter.com/dwcheatham), GitHub's Chief of Staff to the CEO, to explore AI's impact on open source. I hope to “see” you there! Got questions? Drop them below & get answers live!

📆 Friday, May 17, 2024, 10AM PT
📺 [gh.io/friday](https://gh.io/friday)
ℹ️ [meetup event](https://www.meetup.com/github/events/300865395/)

**Comment by CopperEagle** [comment]
Thanks for the heads up :+1:  Given all this AI development and hype, it's a wild time for open source for sure...

*ChatGPT 4o entered the chat* :zany_face:

**Reply by AndreaGriffiths11** [reply]
It is @CopperEagle wild!!! in a fascinating, exciting and scary way to me personally.
